app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb

```diff
@@ -48,8 +48,15 @@ def allowed_subnets(_options = {})
   end
 
   def allowed_cloud_volumes(_options = {})
-    ar_volumes = ar_ems.cloud_volumes
+    storage_type = values&.dig(:storage_type, 1)
+
+    ar_volumes = ar_ems.cloud_volumes.select do |cloud_volume|
+      ((cloud_volume['volume_type'] == storage_type) &&
+      (cloud_volume['multi_attachment'] || cloud_volume['status'] == 'available'))
```

Comment:
Couple extra parens here, can just be `cv['volume_type'] == storage_type && (cv['mutli_attachment'] || cv['status'] == 'available')`

Reply:
Is it correct that Ruby doesn't require parenthesis for multi-line statements if you put the operator at the end of the line to be continued? So this is evaluated as a single statement:
```
cloud_volume['volume_type'] == storage_type &&
(cloud_volume['multi_attachment'] || cloud_volume['status'] == 'available')
```

Reply:
Correct ruby does not require parens around a multi-line conditional